pkg/preparer/orchestrate.go

```diff
@@ -18,10 +18,19 @@ type Pod interface {
 	Halt() (bool, error)
 }
 
+type RealityStore interface {
+	Pod(string, string) (*pods.PodManifest, error)
+	SetPod(string, pods.PodManifest) (time.Duration, error)
+}
+type IntentStore interface {
+	RegisterPodService(pods.PodManifest) error
+	WatchPods(string, <-chan struct{}, chan<- error, chan<- pods.PodManifest) error
+}
+
 type Preparer struct {
 	node   string
-	iStore *intent.Store
-	rStore *reality.Store
+	IStore IntentStore
```

Comment:
Oh yeah I'll fix that. At home my tests usually use the `foo_test` package so they can only access exported identifiers.

On a related note, any opinions about which fields should be public and which ones shouldn't? I wasn't really sure when I made the refactor, so I leaned towards not exporting them.